Sources/Components/Button/Button/Button.swift

```diff
@@ -50,6 +50,10 @@ public class Button: UIButton {
         super.setTitleColor(style.textColor, for: .normal)
         super.setTitleColor(style.highlightedTextColor, for: .highlighted)
         super.setTitleColor(style.disabledTextColor, for: .disabled)
+
+        if FinniversKit.isDynamicTypeEnabled {
```

Comment:
why this check? i would imagine the font size wouldn't change if dynamic type is not enabled

Reply:
I was thinking about it. Read more on the docs, the check shouldn't be needed.